Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-card/ContestCard.tsx

```diff
@@ -229,6 +241,7 @@ const ContestCard = (props: IContestCardProps) => {
                         renderContestDetailsFragment(
                             iconNames.remainingTime,
                             `remaining time: ${remainingTimeFormatted}`,
+                            'Contest\'s remaining time',
```

Comment:
No need for the word "Contest" as this is displayed on the contest card.

Reply:
Done.